src/less/star-rating/base/star-rating.less

```diff
@@ -10,3 +10,34 @@ svg.star-rating {
     vertical-align: middle;
     width: 64px;
 }
+
+div.star-rating svg.star-rating-icon {
```

Comment:
Not a huge deal, but to reduce CSS specificity we could drop the first selector, i.e:

```
svg.star-rating-icon {
    fill: none;
    height: 12px;
    width: 12px;
}
```

Same applies to line 21.

Reply:
Yea. I was trying to make a very clear distinction between the 2 methods to avoid confusion because method 1 is based off `svg.star-rating` and I thought `svg.star-rating-icon` might muddy the waters a bit. Ok. I will remove the `div`.

Reply:
Yep, tbh I could go either way. Sometimes the readibility/clarity is better than specificity concerns.